django/core/management/commands/dumpdata.py

```diff
@@ -6,6 +7,39 @@
 from django.core.management.utils import parse_apps_and_model_labels
 from django.db import DEFAULT_DB_ALIAS, router
 
+try:
+    import bz2
+    has_bz2 = True
+except ImportError:
+    has_bz2 = False
+
+try:
+    import lzma
+    has_lzma = True
+    lzma_kwarg = {'format': lzma.FORMAT_ALONE}
```

Comment:
Do we need a separate variable? I would include it directly in the `compression_formats`:
```python
compression_formats['lzma'] = (lzma.open, {'format': lzma.FORMAT_ALONE}, mode)
```

Reply:
Fixed